design/multiple-storage-types-support.md

```diff
@@ -288,6 +312,14 @@ type Device struct {
 After Kubernetes has done the general deserialization of the `StorageScopeSpec` into a strong type with weakly typed maps of backend specific config properties, the Ceph operator could easily convert this map into a strong config type that is has knowledge of.
 Other backend operators could do a similar thing for their node/device level config.
 
+#### Placement, Resources, Network
+
+As previously mentioned, the `rook.io` API group will also define some other useful `*Spec` types:
```

Comment:
There won't be any crds in this api group, right?

Reply:
not yet :-)